bundles/org.openhab.binding.opengarage/src/main/java/org/openhab/binding/opengarage/internal/OpenGarageWebTargets.java

```diff
@@ -45,19 +46,17 @@ public ControllerVariables getControllerVariables() throws OpenGarageCommunicati
 
     public void setControllerVariables(String request) throws OpenGarageCommunicationException {
         logger.debug("Received request: {}", request);
-          if (request.equals("open")) {
-              logger.debug("Received request to open door");
-              String queryParams = "&open=1";
-              invoke(changeControllerVariablesUri, queryParams);
-          } else if (request.equals("close")) {
-              logger.debug("Received request to close door");
-              String queryParams = "&close=1";
-              invoke(changeControllerVariablesUri, queryParams);
-          } else if (request.equals("click")) {
-              logger.debug("Received request to click remote button");
-              String queryParams = "&click=1";
-              invoke(changeControllerVariablesUri, queryParams);
-          }
+        String queryParams = null;
+        if (request.equals(OpenGarageCommand.OPEN.getValue())) {
```

Comment:
Once the request is an enum, then you can do a switch statement on it like this:
```suggestion
        switch(request){
             case OPEN:
```

Reply:
Fixed in next commit